src/layman_settings_util.py

```diff
@@ -16,3 +17,17 @@ def read_clients_dict_from_env():
         })
         idx += 1
     return client_dicts
+
+
+def validate_layman_role_service_uri(layman_role_service_uri_str):
+    env_name = 'LAYMAN_ROLE_SERVICE_URI'
+    uri = parse.urlparse(layman_role_service_uri_str)
+    exp_uri_schemes = {'postgresql', 'postgres'}
+    assert uri.scheme in {'postgresql', 'postgres'}, \
```

Comment:
`assert uri.scheme in {'postgresql', 'postgres'}, \` -> `assert uri.scheme in exp_uri_schemes, \`